cmd/sqlflowserver/main_test.go

```diff
@@ -350,6 +349,21 @@ func TestEnd2EndMySQL(t *testing.T) {
 	t.Run("CaseTrainFeatureDerivation", CaseTrainFeatureDerivation)
 
 	t.Run("CaseShowTrain", CaseShowTrain)
+
+	// Cases for diagnosis
+	t.Run("CaseDiagnosisMissingModelParams", CaseDiagnosisMissingModelParams)
+}
+
+func CaseDiagnosisMissingModelParams(t *testing.T) {
+	a := assert.New(t)
+	trainSQL := `SELECT * FROM iris.train TO TRAIN DNNClassifier WITH
+  model.n_classes = 3,
+  train.epoch = 10
+COLUMN sepal_length, sepal_width, petal_length, petal_width
+LABEL class
+INTO sqlflow_models.my_dnn_model;`
+	_, _, _, err := connectAndRunSQL(trainSQL)
+	a.Contains(err.Error(), "model attribute missing 1 required positional argument: 'hidden_units'")
```

Comment:
This not user-friendly enough, `positional arguments` is a Python terminology, and `model attribute` is inconsistent with the following `argument`, an ideal diagnostics message should be at least something like:
```
DNNClassifier missing 1 required attribute: model.hidden_units (type: integer list)"
```

Reply:
Diagnosis argument type error would be more complex, will implement it in the next PR.